src/pretix/base/payment.py

```diff
@@ -145,6 +153,15 @@ def settings_content_render(self, request: HttpRequest) -> str:
         pass
 
     @property
+    def invoice_text(self) -> str:
```

Comment:
Wouldn't it be nice to give the provider even more flexbility by not using a property and passing the ``Order`` object in?